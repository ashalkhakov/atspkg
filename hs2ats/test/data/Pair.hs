module Pair where

data Pair = Pair { first :: Int, second :: Int }
