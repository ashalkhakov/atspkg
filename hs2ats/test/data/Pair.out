vtypedef pair = @{ first = int, second = int }
